dotnet/samples/KernelSyntaxExamples/Example48_GroundednessChecks.cs

```diff
@@ -95,15 +95,15 @@ public static async Task GroundednessCheckingAsync()
         Console.WriteLine("======== Extract Entities ========");
         Console.WriteLine(extractionResult);
 
-        variables[KernelArguments.InputParameterName] = extractionResult;
+        variables[reference_check.Metadata.Parameters[0].Name] = extractionResult;
```

Comment:
Can we just say `variables["input"]` to keep the example and syntax simple? Same in other places.